packages/rum-recorder/src/types.ts

```diff
@@ -29,42 +29,24 @@ export type CreationReason =
   | 'before_unload'
   | 'visibility_hidden'
 
-export type RawRecord =
-  | DomContentLoadedRecord
-  | LoadedRecord
-  | FullSnapshotRecord
-  | IncrementalSnapshotRecord
-  | MetaRecord
-  | CustomRecord
-  | FocusRecord
-  | ViewEndRecord
+export type RawRecord = FullSnapshotRecord | IncrementalSnapshotRecord | MetaRecord | FocusRecord | ViewEndRecord
 
 export type Record = RawRecord & {
   timestamp: number
   delay?: number
 }
 
 export enum RecordType {
-  DomContentLoaded,
-  Load,
+  _0, // previously DomContentLoaded
+  _1, // previously Load
   FullSnapshot,
   IncrementalSnapshot,
   Meta,
-  Custom,
+  _2, // previously Custom
   Focus,
   ViewEnd,
```

Comment:
You could use

```
FullSnapshot = 2,
IncrementalSnapshot = 3,
Meta = 4,
Focus = 6,
ViewEnd = 7
```

instead of setting unused enum variants.